#!/bin/sh /usr/share/dpatch/dpatch-run
## 104-magic-remove-cgm.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: FIXME

@DPATCH@

diff -Naurp file-4.23.orig/magic/Magdir/images file-4.23/magic/Magdir/images
--- file-4.23.orig/magic/Magdir/images	2007-05-03 15:11:33.000000000 +0000
+++ file-4.23/magic/Magdir/images	2008-01-12 16:28:03.000000000 +0000
@@ -130,8 +130,8 @@
 # CGM image files
 0	string		BEGMF		clear text Computer Graphics Metafile
 # XXX - questionable magic
-0	beshort&0xffe0	0x0020		binary Computer Graphics Metafile
-0	beshort		0x3020		character Computer Graphics Metafile
+#0	beshort&0xffe0	0x0020		binary Computer Graphics Metafile
+#0	beshort		0x3020		character Computer Graphics Metafile
 
 # MGR bitmaps  (Michael Haardt, u31b3hs@pool.informatik.rwth-aachen.de)
 0	string	yz	MGR bitmap, modern format, 8-bit aligned
@@ -510,7 +510,7 @@
 0	string	AT&TFORM		DjVu Image file
 
 # From: Jason Bacon <bacon@smithers.neuro.mcw.edu>
-0	beshort	0x3020			character Computer Graphics Metafile
+#0	beshort	0x3020			character Computer Graphics Metafile
 
 # From Marc Espie
 0	lelong	20000630		OpenEXR image data
